pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.509.2</version>
+        <version>1.588</version>
```

Comment:
The change still uses the new core version. I'll decrease the dependency to original version and check if everything goes fine